src/adapters/errors.ts

```diff
@@ -0,0 +1,8 @@
+import { NotFoundError } from '@dcl/platform-server-commons'
+
+export class CommunityNotFoundError extends NotFoundError {
```

Comment:
not a strong opinion but i'd try to avoid managing flows through exceptions.

Maybe we can have this error to be a HTTP returning type that we can re-use in our handlers when we detect that the requesting community does not exists?

Reply:
I considered creating a new exception, knowing we use the error handler that catches NotFoundError to return the proper 404 response.